### PR TITLE
chore: refactor body of resolveDimensions

### PR DESCRIPTION
**Pull request by frederickfogerty**
This is part of the Gatsby fp-ts refactor.

### CODE REVIEW

**Comment by frederickfogerty** [comment]
Current dependencies on/for this PR:
* next
  * **PR #190** <a href="https://app.graphite.dev/github/pr/imgix/gatsby/190" target="_blank">Graphite</a> 

This comment was auto-generated by [Graphite](https://app.graphite.dev/github/pr/imgix/gatsby/196?utm_source=stack-comment).

File: src/modules/gatsby-plugin/createImgixGatsbyImageDataFieldConfig.ts [hunk 1]
```diff
@@ -89,8 +88,8 @@ const resolveGatsbyImageData = <TSource>({
     )(
       resolveDimensions({
         url,
-        manualHeight: O.fromNullable(manualHeight),
-        manualWidth: O.fromNullable(manualWidth),
+        manualHeight: manualHeight,
+        manualWidth: manualWidth,
```
Comment (luqven) [inline src/modules/gatsby-plugin/createImgixGatsbyImageDataFieldConfig.ts:92]: nit: do we need to k:v here? Couldn't this just be
```suggestion
        manualHeight,
        manualWidth,
```

**Review by luqven — APPROVED** [review]
LGTM 👍🏼

File: src/modules/gatsby-plugin/createImgixGatsbyImageDataFieldConfig.ts (hunk 1 above)
Comment (frederickfogerty) [reply src/modules/gatsby-plugin/createImgixGatsbyImageDataFieldConfig.ts:92]: Thanks, addressed in a follow-up PR!

**Pull request merged**